keeper/keeper.go

```diff
@@ -46,6 +47,7 @@ func NewKeeper(
 	sk *stakingkeeper.Keeper,
 	slk SlashingKeeper,
 	bk BankKeeper,
+	ak AccountKeeper,
```

Comment:
This is only for test? if so we should have a SetTestAccountKeeper() instead a NewKeeper(...) signature breaking change here.

Reply:
[d4eea29](https://github.com/strangelove-ventures/poa/pull/186/commits/d4eea298b33151028c61710042dfaf0a5ba52f8b)